content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -355,7 +355,7 @@ As soon as the control plane is available, kubeadm executes following actions:
 
 Please note that:
 
-1. Mark control-plane phase phase can be invoked individually with the [`kubeadm init phase mark-control-plane`](/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-mark-master) command
+1. Mark control-plane phase phase can be invoked individually with the [`kubeadm init phase mark-control-plane`](/docs/reference/setup-tools/kubeadm/kubeadm-init-phase/#cmd-phase-mark-control-plane) command
```

Comment:
fix?
Mark control-plane phase phase can be invoked

Reply:
The substitutions look okay. I question whether this is correct:
`Mark control-plane phase phase can be invoked`?
Should this read,
`The control-plane phase "init phase" can be invoked`? OR
`The control plane "init phase" can be invoked`?

Reply:
In another PR I might tweak this to, eg, 
_You can manually invoke the init phase `mark-control-plane` &hellip;_

Reply:
The second 'phase' looks like a typo in the original.

Reply:
Yes. Can you patch up this typo?